### PR TITLE
[SWAGGER] Document Transactions endpoints

### PR DESCRIPTION
**Pull request by ledelaespriella**
## Jira Ticket

- [WA-41](https://n2-alkemy-sku-31-10.atlassian.net/jira/software/projects/WA/boards/1?assignee=6183439e26a54000711fbace&selectedIssue=WA-41)

## Description
AS: member of project
I Want: see how to use endpoints of transactions
To: correctly apply each of the endpoints

## Evidence:

Captura de Pantalla 2022-11-11 a la(s) 3 02 45 p m

## GET

Captura de Pantalla 2022-11-11 a la(s) 3 03 18 p m
Captura de Pantalla 2022-11-11 a la(s) 3 03 36 p m

## POST
Captura de Pantalla 2022-11-11 a la(s) 3 04 15 p m
Captura de Pantalla 2022-11-11 a la(s) 3 05 09 p m

### Whit bad request:

Captura de Pantalla 2022-11-11 a la(s) 3 06 42 p m

## GET BY ID

Captura de Pantalla 2022-11-11 a la(s) 3 09 41 p m
Captura de Pantalla 2022-11-11 a la(s) 3 09 55 p m

### Whit bad request:

Captura de Pantalla 2022-11-11 a la(s) 3 10 23 p m

## PUT

Captura de Pantalla 2022-11-11 a la(s) 3 10 45 p m
Captura de Pantalla 2022-11-11 a la(s) 3 11 02 p m

### Whit bad request:

Captura de Pantalla 2022-11-11 a la(s) 3 11 46 p m

## DELETE

Captura de Pantalla 2022-11-11 a la(s) 3 12 07 p m

### Whit bad request:

Captura de Pantalla 2022-11-11 a la(s) 3 12 24 p m

### CODE REVIEW

File: routes/transactions.router.js [hunk 1]
```diff
@@ -48,20 +51,251 @@ const deleteValidation = require("../schemas/transaction/deleteTransactionsSchem
...
+ *             format: int64
+ *             example: 1
+ *        - name: page
+ *          in: query
+ *          description: Filter by page.
+ *          schema:
+ *             type: integer
+ *             format: int64
+ *             example: 1
+ *     responses:
```
Comment (germanWalton) [inline routes/transactions.router.js:76]: You must add status 400 as response too. If you send and invalid dataType in the query params you will get a Bad Request.

File: routes/transactions.router.js [hunk 2]
```diff
@@ -48,20 +51,251 @@ const deleteValidation = require("../schemas/transaction/deleteTransactionsSchem
...
+ *       '500':
+ *         description: Internal Server Error
+ *   delete:
+ *     summary: Delete Transactions by Id
+ *     tags: [Transactions]
+ *     parameters:
+ *        - name: id
+ *          in: path
+ *          required: true
+ *          description: ID of Transactions to return.
```
Comment (germanWalton) [inline routes/transactions.router.js]: It must be ID of the transaction to delete

File: routes/transactions.router.js [hunk 3]
```diff
@@ -48,20 +51,251 @@ const deleteValidation = require("../schemas/transaction/deleteTransactionsSchem
...
+ *       '500':
+ *         description: Internal Server Error
+ *   put:
+ *     summary: Edit Transactions by Id
+ *     tags: [Transactions]
+ *     parameters:
+ *        - name: id
+ *          in: path
+ *          required: true
+ *          description: ID of Transactions to return.
```
Comment (germanWalton) [inline routes/transactions.router.js]: It must be ID of the transaction to edit

**Review by germanWalton — CHANGES_REQUESTED** [review]

**Comment by germanWalton** [comment]
Check why the endpoints return a validation alert when you pass an incorrect field. It must return a response body with the correct status

**Review by germanWalton — APPROVED** [review]

**Review by AgustinCanteros — APPROVED** [review]

**Pull request merged**